doc/sphinx-guides/source/developers/timers.rst

```diff
@@ -0,0 +1,19 @@
+==========
+EJB Timers
+==========
+
+As described in :doc:`../admin/timers`, Dataverse uses EJB timers for scheduled jobs. This section is about the
+techniques used for scheduling.
+
+* :doc:`../admin/metadataexport` is done via ``@Schedule`` annotation on ``OAISetServiceBean.exportAllSets()`` and
```

Comment:
My main question/source of confusion about this PR so far: I'm not seeing the @Schedule annotation mentioned here in either OAISetServiceBean or DatasetServiceBean in your branch. Am I missing something?

Reply:
I just pushed actual codework on this. Until now, this was only documentation work, as @pdurbin suggested documentation-driven-development... :wink: